packages/mobile/src/invite/RegulatoryTerms.tsx

```diff
@@ -0,0 +1,125 @@
+import Button, { BtnTypes } from '@celo/react-components/components/Button'
```

Comment:
I think it was a mistake to put JoinCelo.tsx in the `invite` folder, not sure when that happened. I'd rather not compound that though. Can you put this is a different folder?

Reply:
Im not seeing an obvious place. there used to be a nux folder, something like that would make most sense to me

Reply:
Or maybe the account folder would be a good spot to move both this and Join to?

ha noticing there is an Invite component in the account folder.

Reply:
Yeah that Invite component should here and these two nux screens should be in a folder like `registration` or something

Reply:
moved those two to regitration